src/app/pages/applications/dialogs/chart-events/chart-events-dialog.component.html

```diff
@@ -3,11 +3,22 @@
     <div class="logo">
       <img [src]="catalogApp.chart_metadata.icon" [src-fallback]="imagePlaceholder"/>
     </div>
-    <div fxLayout="row" fxLayoutAlign="start center">
-      <strong class="chart-name">{{ catalogApp.name }}</strong>
-      <span class="version"> {{ catalogApp.human_version }}</span>
-      <span class="value">{{ appStatus() | translate }}</span>
-      <mat-icon *ngIf="getUpdateVersionTooltip()" class="version-tooltip" matTooltip="{{ getUpdateVersionTooltip() }}">info</mat-icon>
+    <div fxLayout="column" fxLayoutAlign="center flex-start">
+      <div fxLayout="row" fxLayoutAlign="start center">
+        <strong class="chart-name">{{ catalogApp.name }}</strong>
+      </div>
+      <div fxLayout="row" fxLayoutAlign="flex-start center">
+        <span class="top-label">{{ appStatus() | translate }}</span>
+        <mat-icon *ngIf="getUpdateVersionTooltip()" class="version-tooltip" matTooltip="{{ getUpdateVersionTooltip() }}">info</mat-icon>
+      </div> 
+      <div fxLayout="row" fxLayoutAlign="flex-start center">
+        <span class="top-label">{{ 'Container Version' | translate }}:</span>
+        <span class="top-value"> {{ catalogApp.chart_metadata.appVersion }}</span>
+      </div>  
+      <div fxLayout="row" fxLayoutAlign="flex-start flex-start">
+        <span class="top-label">{{ 'App Version' | translate }}:</span>
+        <span class="top-value"> {{ catalogApp.version }}</span>
```

Comment:
Old string: 1.24.1.4931_1.6.8
New string: Container Version:1.24.1.4931 App Version:1
Shouldn't app version be 1.6.8?

Reply:
Shoozle, thats indeed wrong. Picked the wrong version var here it seems, thanks for the headsup!